hplsql/src/main/java/org/apache/hive/hplsql/functions/FunctionDatetime.java

```diff
@@ -166,30 +161,6 @@ void toTimestamp(HplsqlParser.Expr_func_paramsContext ctx) {
       evalNull();
     }
   }
-  
-  /**
-   * FROM_UNIXTIME() function (convert seconds since 1970-01-01 00:00:00 to timestamp)
-   */
-  void fromUnixtime(HplsqlParser.Expr_func_paramsContext ctx) {
-    int cnt = BuiltinFunctions.getParamCount(ctx);
-    if (cnt == 0) {
-      evalNull();
-      return;
-    }
-    long epoch = evalPop(ctx.func_param(0).expr()).longValue();
-    String format = "yyyy-MM-dd HH:mm:ss";
-    if (cnt > 1) {
-      format = evalPop(ctx.func_param(1).expr()).toString();
-    }
-    evalString(new SimpleDateFormat(format).format(new Date(epoch * 1000)));
```

Comment:
Previously, this was using the SimpleDateFormat.  Based on this change, since this function will be forwarded to Hive's built-in function, that one is now defaulting to DateTimeFormatter.  We would want to document the change in behavior for the HPL/SQL function.